ColorPatterns/Scenes/StartMenu/StartMenuController.swift

```diff
@@ -38,13 +40,14 @@ final class StartMenuController: UIViewController {
 private extension StartMenuController {
 
     private func setupDifficultyButtonColor(button: Int) {
+        // TODO: crash if I do:  setupDifficultyButtonColor(button: 52) - this should be forbidden
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Scenes/StartMenu/StartMenuController.swift#L43"></span>TODOs should be resolved (crash if I do:  setupDifficult…).<br />
<code>todo</code> <code>StartMenuController.swift:43</code></td>
    </tr>
  </tbody>
</table>